software/clipbucket.yml

```diff
@@ -0,0 +1,12 @@
+name: "ClipBucket"
+website_url: "https://clipbucket.oxygenz.fr/en/"
+source_code_url: "https://github.com/MacWarrior/clipbucket-v5"
+description: "Start your own Video Sharing website (YouTube/Netflix Clone) in a matter of minutes."
+licenses:
+  - All Rights Reseved
```

Comment:
I don't think this is accurate

https://github.com/MacWarrior/clipbucket-v5/blob/master/LICENSE